.travis.yml

```diff
@@ -7,14 +7,19 @@ os:
 compiler:
   - gcc
   - clang
-  
+
+env:
+  - USE_CMAKE=YES
+  - USE_CMAKE=NO
+
 addons:
   apt:
     packages:
       - texinfo
 
 script:
-  - autoreconf -i
-  - ./configure
+  - mkdir build
+  - cd build
+  - ../scripts/run_config.sh
```

Comment:
I can do that.  

At:
https://groups.google.com/forum/#!topic/travis-ci/uaAP9zEdiCg
one of the travisci founders says:
> The recommended way to do this is to pull out the commands into a shell script and have that be run from your .travis.yml instead.

However, it wasn't quite as ugly to inline as I expected, lets see if it passes.

I'm still planning to rebase/squash after #29 is merged.  If it isn't merged, then more changes will be required to omit cmake testing on osx.